### ISSUE TITLE
multimedia on install

### ISSUE THREAD

**Issue by lxle** [stacefauske/bodhibuilder_2.2.x#7]
this might have something to do with how the distro i spun is setup, but there have been reports that when users tick the 'download multimedia' option on the installer before installing ends up with a malformed lined in the sources.list after install.

things i noticed was, regardless if the 'partner' repository was checked or not it was never included with the sources list after any build. I assumed it was the 'multimedia' tick box and assumed that was on purpose.

in other words, the tick of 'multimedia' causes a malformed sources file and it might have something to do with 'partner' never being included in the build....

anyway to 'grey' that option out on the installer that you know of?

**Comment by lxle** [comment]
i've been testing it for the better part of the evening and it seems to be an isolated incident , i'm closing this thread, working fine.

**Comment by stacefauske** [comment]
Good to hear. Thanks
